base/sim/FairVolume.h

```diff
@@ -27,57 +27,51 @@ class FairVolume : public TNamed
 {
   public:
     FairVolume();
-    FairVolume(TString name, Int_t id = 0, Int_t detid = 0, FairModule* fMod = 0);
-    virtual ~FairVolume();
-
-    //    const char*  GetName() { return fName.Data();}
-    //    TString getName() { return fName;}
+    FairVolume(TString name, Int_t id = 0, Int_t ModId = 0, FairModule* fMod = nullptr);
```

Comment:
@fuhlig1 Can you please also comment on `fModId`?

The default `()` constructor sets it to `-1` (that's why I have set the member initializer to that value), but the default argument for this constructor also has a `0`?